### PR TITLE
Add new telemetry feature

### PR DESCRIPTION
**Pull request by lilyjazz**
add new telemetry feature.

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@lilyjazz This PR is based on release-5.0 but the v5.0 box in the description part is not ticked, which is confusing. Please specify which versions are applicable. Thanks!

**Comment by lilyjazz** [comment]
> @lilyjazz This PR is based on release-5.0 but the v5.0 box in the description part is not ticked, which is confusing. Please specify which versions are applicable. Thanks!

Sorry, i chose a wrong option. This feature is only available in v5.0.2 and next.

**Comment by TomShawn** [comment]
> > @lilyjazz This PR is based on release-5.0 but the v5.0 box in the description part is not ticked, which is confusing. Please specify which versions are applicable. Thanks!
> 
> Sorry, i chose a wrong option. This feature is available in v5.0.2 and next.

Any technical review is needed or not?

**Comment by lilyjazz** [comment]
> > > @lilyjazz This PR is based on release-5.0 but the v5.0 box in the description part is not ticked, which is confusing. Please specify which versions are applicable. Thanks!
> > 
> > 
> > Sorry, i chose a wrong option. This feature is available in v5.0.2 and next.
> 
> Any technical review is needed or not?

I think xukai can review this pr.

File: telemetry.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ TiDB、TiUP 及 TiDB Dashboard 默认会收集使用情况信息，并将这些
 
 - 随机生成的遥测标示符
 - 集群的部署情况，包括各个组件所在的硬件信息（CPU、内存、磁盘）、组件版本号、操作系统版本号等
+- 系统的请求状态，例如请求次数、持续时长等
```
Comment (kaixu120811) [inline telemetry.md:24]: 例如查询请求次数

File: telemetry.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ TiDB、TiUP 及 TiDB Dashboard 默认会收集使用情况信息，并将这些
 
 - 随机生成的遥测标示符
 - 集群的部署情况，包括各个组件所在的硬件信息（CPU、内存、磁盘）、组件版本号、操作系统版本号等
+- 系统的查询请求状态，例如查询请求次数、持续时长等
```
Comment (TomShawn) [inline telemetry.md:24]:
```suggestion
- （自 v5.0.2 起）系统的查询请求状态，例如查询请求次数、持续时长等
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
@kaixu120811 PTAL again, thanks

**Review by kaixu120811 — APPROVED** [review]

**Pull request merged**